.travis.yml

```diff
@@ -1,4 +1,4 @@
-sudo: required
+sudo: false
```

Comment:
Is this required from the other changes?

Reply:
Whoops - nope, it was a leftover from when Travis `sudo: required` builds were broken the other night.